packages/lib-utils/src/index.ts

```diff
@@ -28,7 +29,7 @@ export {
   K8sResourceListOptions,
   K8sResourceListResult,
 } from './k8s/k8s-resource';
-export { getK8sResourceURL } from './k8s/k8s-utils';
+export { getK8sResourceURL, getActiveWorkspace, setActiveWorkspace } from './k8s/k8s-utils';
```

Comment:
I'm not entirely convinced `workspaces` are a K8s concept, but I do think this is the best place for now to put these utils/hooks. We can revisit in the future 👍